spring-ai-core/src/main/java/org/springframework/ai/chat/client/observation/DefaultChatClientObservationConvention.java

```diff
@@ -0,0 +1,144 @@
+/*
+* Copyright 2024 - 2024 the original author or authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* https://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.springframework.ai.chat.client.observation;
+
+import java.util.stream.Collectors;
+
+import org.springframework.ai.chat.client.observation.ChatClientObservationDocumentation.LowCardinalityKeyNames;
+import org.springframework.ai.chat.observation.ChatModelObservationDocumentation;
+import org.springframework.lang.Nullable;
+import org.springframework.util.CollectionUtils;
+
+import io.micrometer.common.KeyValue;
+import io.micrometer.common.KeyValues;
+
+/**
+ * @author Christian Tzolov
+ * @since 1.0.0
+ */
+
+public class DefaultChatClientObservationConvention implements ChatClientObservationConvention {
+
+	public static final String DEFAULT_NAME = "gen_ai.chat.client.operation";
```

Comment:
This might be something more specific to Spring AI

```suggestion
	public static final String DEFAULT_NAME = "spring.ai.chat.client.operation";
```